src/signer_node/message_processor/process_candidateblock.rs

```diff
@@ -194,7 +194,7 @@ mod tests {
     /// * The node's status is Member and its index is 4.
     /// * The round master's index is 0.
     ///
-    /// 1. Send candidateblock message from index 0 node(array index is 1).
+    /// 1. Send candidateblock message from index 0 node(array index is 4).
```

Comment:
The differences between "index" and "array index" confused me.
I think this test case does not need TEST_KEY.pubkeys().
Using sorted public keys makes the purpose of the test case clear.

Reply:
How about changing the order of TEST_KEYS to dictionary order?
This kind of confusion should occur in other test cases but we need to change some other test cases too.

Reply:
if it is ok, I want to create an another PR for it.

Reply:
> if it is ok, I want to create an another PR for it.
Ok.